src/storage/src/render.rs

```diff
@@ -272,7 +272,7 @@ pub fn build_ingestion_dataflow<A: Allocate>(
             let base_source_config = RawSourceCreationConfig {
                 name: format!("{}-{}", connection.name(), primary_source_id),
                 id: primary_source_id,
-                source_exports: description.indexed_source_exports(&primary_source_id),
+                source_exports: description.source_exports.clone(),
```

Comment:
The entire notion and struct for "indexed source exports" is gone. Same theme here, we can use a `GlobalId` anywhere we need to identify a particular output instead of a `usize`